kcfinder/lib/class_image.php

```diff
@@ -116,8 +116,8 @@ final protected function buildImage($image) {
             $img = $image->image;
 
         } elseif (is_array($image)) {
-            list($key, $width) = each($image);
-            list($key, $height) = each($image);
+            $width = $image[0];
+            $height = $image[1];
```

Comment:
Technically this would fail if somebody did something like `$gd = new image_gd(['w' => 200, 'h' => 200]);`. I don't see it used that way anywhere, but it might be safer to use
```php
$width = array_shift($image);
$height = array_shift($image);
```

Reply:
That looks good to me.

Reply:
Great. I see this is your first contribution. Thanks for this. For release notes credit can do you do another PR to add yourself to this file: https://github.com/civicrm/civicrm-core/blob/master/contributor-key.yml

Reply:
jenkins add to whitelist